### DISCUSSION TITLE
Compatibility PrimeOne/Primefaces

### DISCUSSION THREAD

**Discussion by ClementChrt** [primefaces/community#2009] [Figma UI Kit]
Hi everybody ! I already have a licence key of the PrimeOne UI Kit and i just wanna know which version of it is compatible with Primefaces 11.0.0 ? Thanks !

**Comment by onursenture** [comment]
Hi! The PrimeOne UI Kit is designed based on the Lara theme, while PrimeFaces still uses the previous-generation theme called Saga. Therefore, no Figma version is currently compatible with any PrimeFaces version.

**Reply by LunaMooncraft** [reply]
@onursenture I'm a bit confused. PrimeOne UI Kit is based on Lara but PrimeVue v4 beta is based on Aura. When can we expect to have one theme supported for both?

**Reply by onursenture** [reply]
> @onursenture I'm a bit confused. PrimeOne UI Kit is based on Lara but PrimeVue v4 beta is based on Aura. When can we expect to have one theme supported for both?

Hi, It's currently a work in progress. We're updating not only the designs but also the Tokens Studio tokens, so you might expect that it may take some time. We can't share an estimate for now since V4 is still in beta.
